packages/jest-runner/corpus.ts

```diff
@@ -63,4 +62,15 @@ export class Corpus {
 	private replaceSpacesWithUnderscore(s: string): string {
 		return s.replace(/ /g, "_");
 	}
+
+	private createMissingDirectoriesAndSeeds() {
+		fs.mkdirSync(this._inputDirectory, { recursive: true });
+		fs.mkdirSync(this._outputDirectory, { recursive: true });
+		//LibFuzzer directly loads the seed files, even in the regression test
+		// mode, and invokes the fuzz target with them. If other inputs would be
+		// used in the Jest integration, LibFuzzer would not create crashing inputs
+		// for those, hence an empty seed file is created directly in the input
+		// directory.
+		fs.appendFileSync(path.join(this._inputDirectory, "empty"), "");
```

Comment:
I cannot understand the meaning of the second sentence in the comments above, which is probably why I have this question: why are we always creating an `empty` file in the corpus directory, even if there are files present already?

Another point: the function `inputPaths()` should probably exclude (or maybe traverse them too?) sub-directories that it finds. Currently, a sub-directory shows up as a passed test.

Reply:
Good point. Directories are now filtered out.

Reply:
I also changed/extended the mentioned comment. Does it make more sense now?

Reply:
I understand why we generate an `empty` file in case of an empty input directory.
But is there a reason to generate it in non-empty directories? (I guess this is my main point :) )

Currently, the `empty` file will be re-generated even if I delete it.

Reply:
Removed the `empty` file and marked fuzz tests without seeds as `skipped`.